extensions/funqy/funqy-knative-events/runtime/src/main/java/io/quarkus/funqy/runtime/bindings/knative/events/filters/CEAttributeLiteralEqualsFilter.java

```diff
@@ -0,0 +1,84 @@
+package io.quarkus.funqy.runtime.bindings.knative.events.filters;
```

Comment:
I probably ask this, but we can merge those two predicates right?

Reply:
yes, already have it done locally and will push with all other updates